mappings/net/minecraft/client/particle/CrackParticle.mapping

```diff
@@ -1,37 +1,61 @@
 CLASS dbs net/minecraft/client/particle/CrackParticle
-	CLASS dbs$a FactoryItem
+	CLASS dbs$a ItemFactory
 		METHOD a createParticle (Lfs;Lbbp;DDDDDD)Ldco;
-			ARG 1 world
-			ARG 2 x
-			ARG 3 y
-			ARG 5 velocityX
+			ARG 1 parameters
+			ARG 2 world
+			ARG 3 x
+			ARG 5 y
+			ARG 7 z
+			ARG 9 velocityX
+			ARG 11 velocityY
+			ARG 13 velocityZ
 		METHOD a createParticle (Lft;Lbbp;DDDDDD)Ldco;
-			ARG 1 world
-			ARG 2 x
-			ARG 3 y
-			ARG 5 velocityX
-	CLASS dbs$b
+			ARG 1 parameters
+			ARG 2 world
+			ARG 3 x
+			ARG 5 y
+			ARG 7 z
+			ARG 9 velocityX
+			ARG 11 velocityY
+			ARG 13 velocityZ
+	CLASS dbs$b SlimeFactory
```

Comment:
It was suggested to name this SlimeballFactory to be in line with SnowballFactory. SlimeFactory is based on the registry name "item_slime", as is SnowballFactory on "item_snowball".